dao/src/main/java/com/epam/esm/config/DaoConfig.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.esm.config;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import org.springframework.context.annotation.*;
```

Comment:
It's better to not use imports with *

Reply:
Idea automatically unites imports from same packages :c
commit 0e95815

Reply:
This happens because your idea is configured such way, you can change it